src/components/Divider.js

```diff
@@ -20,16 +23,19 @@ export const Divider = ({ thoughtsRanked }) => {
       const maxWidth = _.chain(children).map((child) => {
         if(child.classList.contains('child-divider')) return DIVIDER_PLUS_PX;
         const subs = child.getElementsByClassName('subthought');
-        if(subs.length) return subs[0].offsetWidth+DIVIDER_PLUS_PX;
+        if(subs.length) return (subs[0].offsetWidth+DIVIDER_PLUS_PX);
         else return DIVIDER_PLUS_PX;
       }).max().value();
-      dividerSetWidth.current.style.width = `${maxWidth}px`
+      dividerSetWidth.current.style.width = `${maxWidth>DIVIDER_MIN_WIDTH?maxWidth:DIVIDER_MIN_WIDTH}px`
+      console.log('maxWidth', maxWidth);
     }
   }
 
-  setTimeout(() => {setStyle()}, 300)
+  useEffect(() => {
+    setStyle()
+  });
```

Comment:
Wrapping a function with no arguments in a lambda function is redundant. You can simply pass the function reference: `useEffect(setStyle)`.